### DISCUSSION TITLE
How do I add Keywords with MIDI2LR?

### DISCUSSION THREAD

**Discussion by deadwrong666** [rsjaffe/MIDI2LR#531] [Q&A] [answered]
HI,

I have thousands of pictures sometimes and need to use keywords i believe.
So i want to build them and use the keyboard to just tap the pad and it takes me there to add/create keyword.

IS this possible

**Comment by antzpantz** [comment, marked as answer]
https://github.com/rsjaffe/MIDI2LR/wiki/Keywords
